packages/simple-markdown/src/index.js

```diff
@@ -1863,7 +1876,7 @@ var htmlFor = function (outputFunc /* : HtmlNodeOutput */) /* : HtmlOutput */ {
  * @type {SimpleMarkdown.OutputFor}
  */
 var outputFor = function (
-    /* :: <Rule : Object> */ rules /* : OutputRules<Rule> */,
+    rules: OutputRules<Rule>,
     property /* : $Keys<Rule> */,
     defaultState /* : ?State */,
```

Comment:
Is this plan to convert more of these types to regular annotations in this PR or a followup PR?

Reply:
I can do more in a follow on. Yep